src/modules/SX126x/SX126x.h

```diff
@@ -1047,6 +1043,9 @@ class SX126x: public PhysicalLayer {
 
     int16_t _lastError = RADIOLIB_ERR_NONE;
 
+    // Allow subclasses to define different TX modes
+    unsigned _tx_mode = Module::MODE_TX;
```

Comment:
Please use fully specified types (e.g. uint8_t).

Reply:
Resolved in a64ee65a11b104cec72ae28a512d875660c080a0.